patterns/elastic-fabric-adapter/main.tf

```diff
@@ -127,29 +127,6 @@ module "eks" {
         group_name = aws_placement_group.efa.name
       }
 
-      pre_bootstrap_user_data = <<-EOT
```

Comment:
this is available by default today through 1.25+ and will soon be available through 1.23+